droidlet/interpreter/filter_helper.py

```diff
@@ -230,13 +231,27 @@ def interpret_task_filter(interpreter, speaker, filters_d, get_all=False):
 
 def interpret_dance_filter(interpreter, speaker, filters_d, get_all=False):
     F = MemoryFilter(interpreter.memory)
+
     search_data = {}
     triples = []
+    triple_filter = None
     for t in filters_d.get("triples"):
         triples.append(t)
-    search_data["base_table"] = "Dances"
-    search_data["triples"] = triples
-    F.append(BasicFilter(interpreter.memory, search_data))
+    if triples:
```

Comment:
not sure I understand why we have this snippet here. Looks like triples isn't assigned anything here, we won't even enter the code block, right ?

Reply:
the code block is switched on filters_d.get("triples"), and the triples start empty.  if the filter dict has triples, they will be appended to the list named "triples", and that list will be converted to a BasicSearch Filter.  I think the code is correct as it is, but more wordy than it needs to be (could just use filters_d["triples"] in line 242; but I just wanted to match the structure of the block at 249

Reply:
Ah nvm I missed this piece:
```
for t in filters_d.get("triples"):
        triples.append(t)
```
where we change triples